src/tabbar.cpp

```diff
@@ -122,6 +123,17 @@ WebView* TabBar::createNewTab(bool setCurrent)
                 QUITIFNOTCURRENT(webView);
                 emit webActionEnabledChanged(QWebEnginePage::Forward, webView->isWebActionEnabled(QWebEnginePage::Forward));
             });
+    connect(webView->page(), &QWebEnginePage::linkHovered, this, 
+            [=](const QString& url) {
+                auto tabbar = KiwixApp::instance()->getTabWidget();
+                if (url.isEmpty()) {
+                    QToolTip::hideText();
+                } else {
+                    auto height = tabbar->currentWidget()->height() + 1;
+                    auto pos = tabbar->mapToGlobal(QPoint(-3, height));
```

Comment:
I have to adjust the position of the tooltip here to correctly fit the side of the window when there will be no more status bar (if it's still relevant https://github.com/kiwix/kiwix-desktop/issues/71).

Reply:
I think #71 is still relevant. Please do a PR for this (you've pushed the branch but not created a pr)

Reply:
I already created a pr for this but I closed it when I thought that I have just to have it. I don't know if it's possible to reopen this pr https://github.com/kiwix/kiwix-desktop/pull/233 ?

Reply:
Are you hiding the statusbar here (as said in #71) ?

Reply:
not anymore